Source/WebInspectorUI/UserInterface/Views/NetworkTabContentView.js

```diff
@@ -120,7 +120,7 @@ WI.NetworkTabContentView = class NetworkTabContentView extends WI.TabContentView
 
     dropZoneShouldAppearForDragEvent(dropZone, event)
     {
-        return event.dataTransfer.types.includes("Files");
+        return event.dataTransfer.types.includes("Files") && !event.dataTransfer.types.includes("text/uri-list");
```

Comment:
i think we should split this into a separate PR as im not entirely sure we want to disallow this (i.e. what if the URI points to JSON?)